.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,8 @@
 
 Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
+> **_NOTE_** Modifications to charmcraft files like `config.yaml`, `actions.yaml` and `metadata.yaml` may include documentation changes that appear directly on the charmhub listing. Therefore, the type of change field should include `documentation update` if your PR contains such changes.
```

Comment:
"...may include changes that surface on a charm's Charmhub listing as documentation (e.g. option and action descriptions)."

I'm not certain whether `metadata.yaml` fits into this category, other than the fact that it may point to a charm's default landing (docs) page (via the `docs` key).

Reply:
I would not want to miss a `docs` key update 😳

Reply:
And I think it also contains the license, Summary, Description etc ? (are these values not used in CharmHub ?)